pep-9999.rst

```diff
@@ -0,0 +1,259 @@
+PEP: 9999
```

Comment:
```suggestion
PEP: 609
```

Reply:
Question -- any reason to not make this a 2 digit PEP? IMO that'd make it more discoverable, and it's more similar in intent with "Python Language Governance" than like, CPython's release schedules or manylinux PEPs.

That said, maybe the ship has sailed on this given that it's already been merged.